agent/qrexec-agent.c

```diff
@@ -139,9 +132,8 @@ static struct pam_conv conv = {
  * If dom0 sends overly long cmd, it will probably crash qrexec-agent (unless
  * process can allocate up to 4GB on both stack and heap), sorry.
  */
-_Noreturn void do_exec(char *cmd)
+_Noreturn void do_exec(char *cmd, const char *user)
```

Comment:
unrelated cleanup, I think. Won't mark the related changes.

Reply:
It actually isn’t ― it is part of the parsing changes.